doc/source/reference/routines.other.rst

```diff
@@ -35,3 +36,26 @@ NumPy version comparison
    :toctree: generated/
 
    lib.NumpyVersion
+
+NumPy header files location
+----------------------------
+.. autosummary::
+   :toctree: generated/
+   
+   get_include
+
+Deprecation warning
+-------------------
+.. autosummary::
+   :toctree: generated/
+
+   deprecate
+
+Details of used variables
+-------------------------
+.. autosummary::
+   :toctree: generated/
+ 
+   who
```

Comment:
We should mention `disp` here, even though it's deprecated - that way a google search will show that it's deprecated.

Perhaps "matlab-like functions" would be a better title for these two.

Reply:
got it